config/settings.yml

```diff
@@ -34,6 +34,10 @@ mobile_application_platform:
   client_key_path: spec/fixtures/mobile_application_platform/oauth.key
   sign_up_service_url: https://cerner.apps-staging.va.gov
   sign_up_service_provisioning_api_key: abcd1234
+  sts:
+    mock: true
+  sus:
+    mock: true
```

Comment:
We'll need to add these to the application_manifests and make sure they are set to `false`.  [vsp-infra-application-manifests](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api)

Reply:
I've got the changes for both of our tickets on the PRs to update the manifests:

- https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2314
- https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2315